src/common/messages.ts

```diff
@@ -49,150 +48,148 @@ export interface VisualizationMessages {
     DetailsView: DetailsViewMessages;
 }
 
+const messagePrefix = 'insights';
+
 export class Messages {
     public static readonly Visualizations: VisualizationMessages = {
         Common: {
-            Toggle: 'insights/visualization/toggle',
-            ScanCompleted: 'insights/visualization/scanCompleted',
-            ScrollRequested: 'insights/visualization/scrollRequested',
+            Toggle: `${messagePrefix}/visualization/toggle`,
```

Comment:
would it be better if we use a function to add prefix to all messages in order to reduce dupilication?

Reply:
Any specific suggestions?